pages/index.vue

```diff
@@ -1879,28 +1877,27 @@ export default {
         return requestString.join("")
       } else if (this.requestType === "cURL") {
         const requestString = []
-        requestString.push(`curl -X ${this.method} \n`)
-        requestString.push(`  '${this.url}${this.pathName}${this.queryString}' \n`)
+        requestString.push(`curl -X ${this.method}`)
+        requestString.push(`  '${this.url}${this.pathName}${this.queryString}'`)
         if (this.auth === "Basic Auth") {
           const basic = `${this.httpUser}:${this.httpPassword}`
           requestString.push(
-            `  -H 'Authorization: Basic ${window.btoa(unescape(encodeURIComponent(basic)))}' \n`
+            `  -H 'Authorization: Basic ${window.btoa(unescape(encodeURIComponent(basic)))}'`
           )
         } else if (this.auth === "Bearer Token" || this.auth === "OAuth 2.0") {
-          requestString.push(`  -H 'Authorization: Bearer ${this.bearerToken}' \n`)
+          requestString.push(`  -H 'Authorization: Bearer ${this.bearerToken}'`)
         }
         if (this.headers) {
           this.headers.forEach(({ key, value }) => {
-            if (key) requestString.push(`  -H '${key}: ${value}' \n`)
+            if (key) requestString.push(`  -H '${key}: ${value}'`)
           })
         }
         if (["POST", "PUT", "PATCH", "DELETE"].includes(this.method)) {
           const requestBody = this.rawInput ? this.rawParams : this.rawRequestBody
-          requestString.push(`  -H 'Content-Length: ${requestBody.length}' \n`)
-          requestString.push(`  -H 'Content-Type: ${this.contentType}; charset=utf-8' \n`)
-          requestString.push(`  -d '${requestBody}' \n`)
+          requestString.push(`  -H 'Content-Type: ${this.contentType}; charset=utf-8'`)
+          requestString.push(`  -d '${requestBody}'`)
         }
-        return requestString.join("").slice(0, -2)
+        return requestString.join(" \\\n")
```

Comment:
A `\\` is additional. Please remove it. Only `\n` is enough.

Reply:
@liyasthomas No, adding a trailing backslash `\\` to each line is the whole point, otherwise you can't just paste-and-execute a multiline shell command.

Reply:
oops. I'll revert my commit.